src/wallet.js

```diff
@@ -0,0 +1,90 @@
+const fs = require('fs');
+const crypto = require('crypto');
+
+let privateKey = null;
+let publicKey = null;
+
+// Mempool and UTXOs simulation
+const mempool = [];
+const utxos = [];
+
+// Generate keys and save to file
+function generateKeys() {
+  const { privateKey: privKey, publicKey: pubKey } = crypto.generateKeyPairSync('rsa', {
+    modulusLength: 2048,
+    publicKeyEncoding: { type: 'spki', format: 'pem' },
+    privateKeyEncoding: { type: 'pkcs8', format: 'pem' },
+  });
+
+  privateKey = privKey;
+  publicKey = pubKey;
+
+  fs.writeFileSync('private_key.pem', privateKey);
+  fs.writeFileSync('public_key.pem', publicKey);

```

Comment:
These 2 steps may fail, could you please add error handling for generating keys?